lighthouse-core/audits/critical-request-chains.js

```diff
@@ -62,7 +64,7 @@ class CriticalRequestChains extends Audit {
       walk(chains, 0);
 
       return CriticalRequestChains.generateAuditResult({
-        rawValue: chainCount.length <= this.meta.optimalValue,
+        rawValue: chainCount <= this.meta.optimalValue,
```

Comment:
wow, this was just broken before. `chainCount` was not an array :\

Not sure if chains vs. depth is better, but if you undo line 53, I think it takes us back to a working audit.